model/flow/dkg.go

```diff
@@ -14,6 +14,9 @@ const (
 	DKGEndStateNoKey
 	// DKGEndStateDKGFailure - the underlying DKG library reported an error.
 	DKGEndStateDKGFailure
+	// DKGEndStateRecovered - this node has recovered its beacon key from a previous epoch.
+	// This occurs only for epochs which are entered through the EFM Recovery process (`flow.EpochRecover` service event).
+	DKGEndStateRecovered
```

Comment:
I find this conceptually confusing, because we are not recovering the DKG. Our goal is to arrive with a valid private threshold key (aka, the Random Beacon key) -- the DKG is only the means to get to this key on the happy path. 

```suggestion
	// RandomBeaconKeyRecovered - this node has recovered its beacon key from a previous epoch.
	// This occurs only for epochs which are entered through the EFM Recovery process (`flow.EpochRecover` service event).
	RandomBeaconKeyRecovered
```